manuscript.Rmd

```diff
@@ -566,7 +619,47 @@ The theoretical risk, however low, must be communicated clearly.
+We find it difficult to content ourselves with vague terms like "control" or "transparency" if they ultimately remain unconnected to how much researchers believe in a theory.
+Within our framework, researchers have the ability to input their assumptions regarding the perspectives of other researchers and calculate the potential impact of their actions on their readership, whether these actions relate to study design, to the preregistration itself, or subsequent deviations from it.
+We put subjective evaluations at the centre of our consideration; we deal explicitly with researchers who are proponents of some theory (they have higher priors for the theory being true), researchers who suspect confounding variables (they assume lower theoretical risk), or those who remain doubtful if everything relevant was reported (they have higher uncertainty about theoretical risk) or even those who place greater value on incongruent evidence then others (they differ in their confirmation function).
+We, therefore, hope to not only provide a rationale for preregistration for those who subscribe to a Bayesian philosophy of science but also a framework to navigate the complicated questions that arise in the practice of preregistration.
+
+At the same time, approaching the evaluation of evidence using a Bayesian formalism is far from novel [e.g., @kuklaClinicalStatisticalTheory1990 @fiedlerWhatConstitutesStrong2017;@sprengerBayesianPhilosophyScience2019].
+To our knowledge, it was not yet applied to the problem of preregistration.
+However, @oberauerAddressingTheoryCrisis2019 made use of the formalism to model the relation between theory, hypothesis, and evidence.
+In the context of this conceptualization, they discussed the usefulness of preregistration, though without applying the formalism there.
+Most importantly, they are rather critical of the idea that preregistration has tangible benefits.
+Instead, they prefer multiverse analyses but contend that those could be preregistered if one fancies it.
+Their reasoning is based on two intuitions about what should *not* influence the evaluation of evidence: temporal order and the mental state of the originator.
+For one, they disregard the temporal order a bit too hastily, as it is a long-standing issue in Bayesian philosophy of science known as the "problem of old evidence" [@chiharaProblemsBayesianConfirmation1987].
+However, we agree that the information flow matters for preregistration, not the temporal order.
+For the other, we argue that the mental state of the originator does matter.
+Suppose there are $k = 1, 2, \ldots, K$ ways to analyze data.
+Where each $k$ has a $P(E_k | \neg H) > 0$.
+If they intend to try each way after another but happen to be "lucky" on the first try and stop, should we then apply $P(\neg E| \neg H) = P(\neg E_1 | \neg H)$ or $P(\neg E| \neg H) = \prod_{k=1}^K P(\neg E_k | \neg H)$?
```

Comment:
I liek this example and agree with it conceptually, however, the formula you give only holds when E_1 ... E_k are independent, which is not the case normally?
So I assume you would write it this way?: (pls also check for latex rendering)
```suggestion
If they intend to try each way after another but happen to be "lucky" on the first try and stop, should we then apply $P(\neg E| \neg H) = P(\neg E_1 | \neg H)$ or $P(\neg E| \neg H) = P(\neg E_1 \land \ldots \land \neg E_k | \neg H)$?
```